Materialize.md

```diff
@@ -0,0 +1,33 @@
+# Materialize
+
+Materialize is a Responsive CSS framework based on Google's [Material Design](https://design.google.com/spec/) Language.
+
+### Getting Started
```

Comment:
@theflametrooper normally you would go in ascending order, Heading 1, 2,3 .. n

The bigger the number the deeper it is into the root header. An article should ideally shoudl have One head 1, and the rest top headings would be heading 2, with subheadings as needed.